prowler/providers/gcp/services/iam/iam_service.py

```diff
@@ -117,12 +117,11 @@ def __get_settings__(self):
                         name=f"projects/{project_id}/accessApprovalSettings"
                     )
                 ).execute()
-                self.settings[project_id].append(
-                    Setting(
-                        name=response["name"],
-                        project_id=project_id,
-                    )
+                self.settings[project_id] = Setting(
```

Comment:
This code was giving `Key Error` when I was testing it, and it does not make sense to do an append because the API always return one item with this call